docs/cmd-docker-doctor.md

```diff
@@ -0,0 +1,21 @@
+# `docker:doctor` command
+
+Diagnose problems with Docker setup and attempt to fix them.
+
+## Usage
+
+```
+dock-cli docker:doctor --dry-run
+dock-cli docker:doctor
+```
+
+## Available options
+
+- `--dry-run` don't attempt to fix the problems, just show problems and suggest
+  appropriate solutions
+
+## What does it do?
+
+- Test whether docker is installed and running.
```

Comment:
For communication purposes I think that's really interesting, and the idea of this tool is to simplify usage of Docker, not to mystify it, so we should explain as much as possible what we do.